lib/options.js

```diff
@@ -20,18 +19,35 @@ const SUPPORTED_VERSIONS = [
     12
 ];
 
+/**
+ * Get the latest ECMAScript version supported by Espree.
+ * @returns {number} The latest ECMAScript version.
+ */
+export function getLatestEcmaVersion() {
+    return SUPPORTED_VERSIONS[SUPPORTED_VERSIONS.length - 1];
+}
+
+/**
+ * Get the list of ECMAScript versions supported by Espree.
+ * @returns {number[]} An array containing the supported ECMAScript versions.
+ */
+export function getSupportedEcmaVersions() {
+    return [...SUPPORTED_VERSIONS];
+}
+
 /**
  * Normalize ECMAScript version from the initial config
  * @param {number} ecmaVersion ECMAScript version from the initial config
```

Comment:
I think this is the right JSDoc syntax for a union type with a literal value?

```suggestion
 * @param {(number|"latest")} ecmaVersion ECMAScript version from the initial config
```